layout_padded/layout_padded.bs

```diff
@@ -803,19 +840,53 @@ template<size_t other_padding_stride, class OtherExtents>
 
 * `other_padding_stride` does not equal `dynamic_extent`,
     
-then the least multiple of `other_padding_stride` greater than or equal to `OtherExtents::static_extent(0)`
+then the least multiple of `other_padding_stride`
+greater than or equal to `OtherExtents::static_extent(0)`
 
-* is representable as a value of type `Extents::index_type`, and
+* is representable as a value of type `Extents::index_type`
+    (*[basic.fundamental]*), and
 
 * equals `Extents::static_extent(0)`.
 
+<i>[Note:</i>
+
+The above Mandates clause and some subclauses of [mdspan] below
+mandate that the least multiple of a `constexpr` `size_t` value $p$
+(such as `other_padding_stride` above)
+greater than or equal to some other nonnegative `constexpr` value $e$ of integral type `U`
+is representable as a value of some (possibly different) integral type `V`,
+where $e$ is always representable as a value of type `V`.
+We explain here how to express this condition as a C++ constant expression.
+
+If $p$ is zero, then $e$ must be zero.  (FIXME)
```

Comment:
We'll need to address this issue that "least multiple of $p$ greater than to equal to $e$" doesn't work if $p$ is zero.

Reply:
Fixed in my most recent commit.